package.json

```diff
@@ -4,27 +4,29 @@
   "description": "erc721.org",
   "main": "main.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "gulp buildclean"
```

Comment:
Is this supposed to be gulp clean? buildclean isn't defined. I think we just leave the error in if we don't have tests at any rate.